index.bs

```diff
@@ -9875,6 +9875,33 @@ Harry Halpin
 for their contributions as our W3C Team Contacts.
 
 
+# Revision History # {#revision-history}
+
+This section is <i>non-normative</i>.
+
+This section contains the substantive changes that have been made to this specification over time.
+
+## Changes since Web Authentication Level 2 [[webauthn-2-20210408]] {#changes-since-l2}
+
+- Two authenticatorData flags assigned: 
+    - [[#sctn-credential-backup]]
+    - [[#sctn-automation-set-credential-properties]]
+- [[#sctn-getClientCapabilities]]
+- Conditional mediation for get: [[#sctn-getAssertion]]
+- Conditional mediation for create: [[#sctn-createCredential]]
+- Added topOrigin: [[#dictionary-client-data]]
+- Create operations in an iframe: [[#sctn-createCredential]]
```

Comment:
I reckon this should also reference section [§5.10. Using Web Authentication within `iframe` elements](https://w3c.github.io/webauthn/#sctn-iframe-guidance); see also https://github.com/w3c/webauthn/issues/2229

Reply:
This would be covered by meta-PR #2230.

Reply:
[Section 5.10. Using Web Authentication within iframe elements](https://www.w3.org/TR/webauthn-2/#sctn-iframe-guidance) existed as-is in [Level 2](https://www.w3.org/TR/webauthn-2/), so it's not a change.  Therefore I don't think it should be added to this list.

Reply:
As is right now, yes, but as noted in issue #2229 this section needs to be updated to reflect that `create()` is now also allowed in cross-origin iframes (and requires the respective feature policy).

Reply:
PR #2231 would make the aforementioned update.